src/components/Images/Images.jsx

```diff
@@ -0,0 +1,20 @@
+import cn from "classnames";
+
+export default function Images({ source, onClick, imagetype }) {
+  const IMAGE_PATH = "/images/";
+  const imgtype = cn(
+    "image",
+    { "user-image": imagetype === "user" },
+    { "gallery-image": imagetype === "gallery" },
+    { "logo-image": imagetype === "logo" }
+  );
+
+  return (
+    <img
+      className={imgtype}
+      onClick={onClick}
+      src={IMAGE_PATH + source}
+      alt=""
+    />
+  );
+}

```

Comment:
```suggestion
export default function Images({ source, onClick, imagetype, alt }) {
  const IMAGE_PATH = "/images/";
  const imgtype = cn(
    "image",
    { "user-image": imagetype === "user" },
    { "gallery-image": imagetype === "gallery" },
    { "logo-image": imagetype === "logo" }
  );

  return (
    <img
      className={imgtype}
      onClick={onClick}
      src={IMAGE_PATH + source}
      alt={alt}
    />
  );
}
```

it's a best practice to pass an alt for all images:

> The ALT text adds a text description to an image on a Web page and should be used for all images, graphical bullets, and graphical horizontal rules. ALT text is accessed by screen reader users to provide them with a text equivalent to images.

https://accessibility.psu.edu/images/imageshtml/#:~:text=The%20ALT%20text%20adds%20a,a%20text%20equivalent%20of%20images.